scripts/check_model.py

```diff
@@ -0,0 +1,57 @@
+"""
+Check for db indexes defined in mapping.py but missing in the database.
+Note: pass '-c /path/to/galaxy.yml' to use the database_connection set in galaxy.yml.
+Otherwise the default sqlite database will be used.
+"""
+import json
+import os
+import sys
+from collections import namedtuple
+
+sys.path.insert(1, os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir, 'lib')))
+
+from sqlalchemy import create_engine, MetaData
+
+from galaxy.model import mapping
+from galaxy.model.orm.scripts import get_config
+
+IndexTuple = namedtuple('IndexTuple', 'table column_names')
+
+
+def tuple_from_index(index):
+    columns = tuple([getattr(index.columns, c).name for c in dir(index.columns) if not c.startswith('__')])
+    if len(columns) == 1:
+        columns = columns[0]
+    return IndexTuple(index.table.name, columns)
+
+
+def find_missing_indexes():
+
+    def load_indexes(metadata):
+        indexes = {}
+        for t in metadata.tables.values():
+            for index in t.indexes:
+                index_tuple = tuple_from_index(index)
+                indexes[index_tuple] = index.name
+        return indexes
+
+    # load metadata from mapping.py
+    metadata = mapping.metadata
+    mapping_indexes = load_indexes(metadata)
+
+    # create EMPTY metadata, then load from database
+    db_url = os.environ.get("GALAXY_CONFIG_OVERRIDE_DATABASE_CONNECTION") or get_config(sys.argv)['db_url']
```

Comment:
```suggestion
    db_url = get_config(sys.argv)['db_url']
```
The override handling is done by get_config.

Reply:
if you want to pass `GALAXY_CONFIG_OVERRIDE_DATABASE_CONNECTION` into tox you need to add this this `passenv` in tox.ini

Reply:
Indeed it does handle it; not at all obvious though. Thanks for both suggestions.
If this is not in developer docs (and I think it's not), I'll add it.